src/components/StatusDashboard/repos_and_bots.jsx

```diff
@@ -0,0 +1,130 @@
+          <table style={{ fontSize: "small" }}>
+            <tbody>
+              <CDNStatus />
+              <WebServices />
+              {urls.repos.badges.map(({ name, ...badge }, index) =>
+                <Badge key={index} {...badge}>{name}</Badge>
+              )}
+            </tbody>
+          </table>
+        </div>
+      </div>
+    </>
+  );
+}
+
+function Badge({ children, link, badge, badgeLink }) {
+  return (
+    <tr>
+      <td>
+        <a href={link} style={{ display: "inline-block", minWidth: "100%" }}>
+          {children}
+        </a>
+      </td>
+      <td style={{ textAlign: "right" }}>
+        <Image alt={`${children} status`} link={badgeLink}>{badge}</Image>
+      </td>
+    </tr>
+  );
+}
+
+function Image({ alt, link, children }) {
+  const [error, setState] = useState(false);
+  const onError = () => setState(true);
+  if (error) return (<>No status available</>);
+  const image = (
+    <img alt={alt} className={styles.badge} onError={onError} src={children} />
+  );
+  return link ? <a href={link}>{image}</a> : image;
+}
+
+function CDNStatus() {
+  const [{ minutes, status }, setState] = useState({ minutes: 0, status: "…" });
+  useEffect(() => {
+    void (async () => {
+      try {
+        const response = await (await fetch(urls.repos.cdn.api)).text();
+        const updated = new Date(response.trim()).getTime();
+        const delta = (new Date()).getTime() - updated;
+        const status = delta < OPERATIONAL_WINDOW ? "operational" :
+          delta < DEGRADED_WINDOW ? "degraded" : "major outage";
+        setState({ minutes: Math.round(delta / 1000 / 60), status });
+      } catch (error) {
+        console.warn(`error loading cdn cloning status`, error);
+      }
+    })();
+  }, []);
+  return (
+    <tr>
+      <td>
+        <a href={urls.repos.cdn.link}
+          style={{ display: "inline-block", minWidth: "100%" }}>
+          CDN cloning
+        </a>
+      </td>
+      <td>
+        <div style={{ display: "block" }} className={
+          styles.status_pill +
+          (status === "operational" ? " " + styles.operational : "") +
+          (status === "degraded" ? " " + styles.degraded : "") +
+          (status === "major outage" ? " " + styles.outage : "")
+        }>{status}</div>
+        <div style={{ fontStyle: "italic", textAlign: "right" }}>
+          (last updated {minutes} min ago)
+        </div>
+      </td>
+    </tr>
+  );
+}
+
+function WebServices() {
+  const [status, setState] = useState("");
+  useEffect(() => {
+    void (async () => {
+      try {
+        const { status } = await (await fetch(urls.repos.services.api)).json();
+        setState(status);
+      } catch (error) {
+        console.warn(`error loading web services status`, error);
+      }
+    })();
+  }, []);
+  return (
+    <tr>
+      <td>
+        <a href={urls.repos.services.link}
+          style={{ display: "inline-block", minWidth: "100%" }}>
+          admin web services
```

Comment:
This should be either `webservices` or `conda-forge-webservices`, to make it match the repo name.